bin/sl-meshctl.txt

```diff
@@ -0,0 +1 @@
+xxx
```

Comment:
didn't get to this yet? should be able to just copy down `sl-pmctl.txt` from strong-pm